client/src/screens/groups.screen.js

```diff
@@ -70,24 +72,27 @@ const formatCreatedAt = createdAt => moment(createdAt).calendar(null, {
   sameElse: 'DD/MM/YYYY',
 });
 
-const Header = ({ onPress }) => (
+const Header = ({ onPress, onPressFind }) => (
   <View style={styles.header}>
-    <Button title={'New Group'} onPress={onPress} />
+    <Button title={'Create New Group'} onPress={onPress} />
+    <Button title={'Find & Join Groups'} onPress={onPressFind} />
   </View>
 );
 Header.propTypes = {
   onPress: PropTypes.func.isRequired,
+  onPressFind: PropTypes.func.isRequired,
 };
 
 class Group extends Component {
   constructor(props) {
     super(props);
-
     this.goToGroup = this.props.goToGroup.bind(this, this.props.group);
   }
 
   render() {
     const { id, name } = this.props.group;
+    const tags = (this.props.group.tags.map((elem) => {return '#'+elem.name})).join(',')
```

Comment:
function

Reply:
how come? its the only placed its used, if I needed it twice I would.